apiserver/facades/controller/crossmodelrelations/crossmodelrelations.go

```diff
@@ -406,6 +406,7 @@ func (api *CrossModelRelationsAPI) WatchRelationChanges(remoteRelationArgs param
 	for i, arg := range remoteRelationArgs.Args {
 		w, changes, err := watchOne(arg)
 		if err != nil {
+			logger.Tracef("not found watching relation %s: %s", arg.Token, errors.ErrorStack(err))
```

Comment:
This error stack and the one in the worker were really useful in working out exactly what wasn't found - so I figured I'd leave them in at trace level.

Reply:
Very much a fan of these!